server/Dockerfile

```diff
@@ -13,6 +13,16 @@ USER jboss
 
 RUN cd /opt/jboss/ && curl -L https://downloads.jboss.org/keycloak/$KEYCLOAK_VERSION/keycloak-$KEYCLOAK_VERSION.tar.gz | tar zx && mv /opt/jboss/keycloak-$KEYCLOAK_VERSION /opt/jboss/keycloak
 
+USER root
+
+#Give correct permissions when used in an OpenShift environment.
+RUN mkdir -p /opt/jboss/keycloak/standalone/log
+RUN mkdir -p /opt/jboss/keycloak/standalone/data
+RUN chown -R jboss:0 /opt/jboss/keycloak/standalone
+RUN chmod -R g+rw /opt/jboss/keycloak/standalone
```

Comment:
No offense, but to me it looks like someone is building a docker images for the first time.